message/models.py

```diff
@@ -0,0 +1,57 @@
+from django.db import models
+from django.utils import timezone
+from dogowner.models import DogOwner
+from daycare.models import DayCare
+
+
+class Message(models.Model):
+
+    author = models.CharField(max_length=200)
+    dogowner_id = models.ForeignKey(DogOwner, on_delete=models.DO_NOTHING)
+    daycare_id = models.ForeignKey(DayCare, on_delete=models.CASCADE)
```

Comment:
I don't think that we should delete the chat in case DogOnwer / DayCare deleted since the other will want to see the chat, We need to delete the messages when both deleted

Reply:
Agree